docs/appguide.rst

```diff
@@ -95,32 +97,40 @@ A backgrounded application enters the ``ACTIVE`` state via a call to
 :py:meth:`~.TemplateApp.foreground`. When it is active the application owns the
 screen and must draw and maintain its user interface.
 
-If the system manager wants to put the watch to sleep then it will tell the
-active application to :py:meth:`~.TemplateApp.sleep`.
-If the application returns True then the application will remain active
-whilst the watch is asleep.
-It will receive no events nor the application tick whilst the system is
-asleep and, instead, must wait for a :py:meth:`~.TemplateApp.wake` notification
-telling the application that the device is waking up and that it may
-update the screen if needed.
-
-If an application does not support sleeping then it can simply not implement
-:py:meth:`~.TemplateApp.sleep` or :py:meth:`~.TemplateApp.wake`.
-In this case the system manager will automatically return to the default
-application, typically the main clock face.
-
-Some applications may support sleeping only under certain circumstances. For
-example a stopwatch may choose to remain active when the watch sleeps only if
-the stopwatch is running.
-This type of application must implement :py:meth:`~.TemplateApp.sleep` and
-return False when it does not want to remain active when the system
-resumes.
+If the screen timeout is near, the system manager will want to turn the screen
+off to save power. It will call the sleep entry point of the currently 
+active application (here, it would be :py:meth:`~.TemplateApp.sleep`).
+Several situations are possible:
+
+* If no sleep() method was implemented in the app, a default sleep()
+  method will be used to turn the screen off, put the application in the
+  background and switch to the default application (usually the main clock
+  application). Turning the screen on will appear as though the user
+  exited the application.
```

Comment:
It would be better to describe this as two possible cases:

1. App implements :py:meth:`~/TemplateApp.sleep()` and returns True
2. Either the app does not implement :py:meth:`~/TemplateApp.sleep()` or the app implements :py:meth:`~/TemplateApp.sleep()` and it returns False.

Also I am not clear why we need to say what happens when we leave the low power state. It is clearer to say what does happen (the application is closed by the system during the system sleep)Does it really appear as though the user exited the application when the ...

Reply:
I just added your text in 78bb996